### ISSUE TITLE
Support for namespaced keywords

### ISSUE THREAD

**Issue by sebastianpoeplau** [layerware/hugsql#56]
As far as I understand, HugSQL doesn't currently support namespace-qualified keywords:

```clojure
> (hugsql.core/sqlvec "SELECT :test/value" {:test/value 42 :test 5})
["SELECT ? /value" 5]
```

(The expected result would be `["SELECT ?" 42]`, I guess.)

I think it would be very helpful to be able to process namespaced keywords, especially with the advent of `clojure.spec`. Are there any known obstacles to handling them? I would try to implement what is needed but wanted to check for known issues or objections first...

**Comment by csummers** [comment]
@sebastianpoeplau I think this is a good idea, and I've known it's coming with spec in Clojure 1.9.

We'll have to look at any possible clashes with `/` in SQL for division.  The result may just be "you must have a space after a keyword to use `/`".  If that's the case, adding `/` as a valid symbol character to the parser is the start for this:

https://github.com/layerware/hugsql/blob/master/hugsql-core/src/hugsql/parser.clj#L29

**Comment by sebastianpoeplau** [comment]
Great, I'll look into it.

**Comment by jbethune** [comment]
Any news on this? Sadly neither YeSQL nor HugSQL support this.

**Comment by sebastianpoeplau** [comment]
The code has been proposed via #57, which for some reason is not linked here...

**Comment by abeluck** [comment]
Is there a workaround for this in 2019? I'm a little surprised, is no one using spec/namespaced keywords and hugsql?

**Comment by csummers** [comment]
This has been merged in and modified to continue to allow the deep-get syntax.  This will ship in 0.5.0 soon.

**Comment by csummers** [comment]
This is released in `0.5.0`.
